lib/geckodriver.js

```diff
@@ -61,6 +61,7 @@ class Geckodriver extends events.EventEmitter {
 
   async start (caps, emitStartingState = true) {
     this.capabilities = _.cloneDeep(caps.desiredCapabilities || caps.capabilities);
+    this.geckodriverExecutable = this.capabilities.geckodriverExecutable || await getGeckodriverForOs();
```

Comment:
It might be useful to log the actual driver location

Reply:
Done! Added to `startGeckodriverSubprocess` on `log.info(`Spawning geckodriver with: ${quote([executable, ...args])}`);`